nightwatch/features/event-reconciliation/reconciled.feature

```diff
@@ -0,0 +1,151 @@
+Feature: Reconciled
+
+  Background:
+    Given I am a logged in user
+    And There are no existing centres
+    And The following centres exist:
+      | name | male_capacity | female_capacity | male_cid_name  | female_cid_name |
+      | one  | 1000          | 10000           | oneman,onebman | onewoman        |
+    Given I am on the wallboard
+
+  Scenario: Reconciled Check In Event and Movement In does not show as Unexpected In and Scheduled In or affect Availability
+    Given The Centre "one" should show the following under "Male":
+      | Contractual Capacity   | 1000 |
```

Comment:
I haven't even considered improving these tests yet.

On Wednesday, 4 May 2016, Chris Nesbitt-Smith <notifications@github.com>
wrote:

> In nightwatch/features/event-reconciliation/reconciled.feature
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6#discussion_r62003591>
> :
>
> > @@ -0,0 +1,151 @@
> > +Feature: Reconciled
> > +
> > +  Background:
> > +    Given I am a logged in user
> > +    And There are no existing centres
> > +    And The following centres exist:
> > +      | name | male_capacity | female_capacity | male_cid_name  | female_cid_name |
> > +      | one  | 1000          | 10000           | oneman,onebman | onewoman        |
> > +    Given I am on the wallboard
> > +
> > +  Scenario: Reconciled Check In Event and Movement In does not show as Unexpected In and Scheduled In or affect Availability
> > +    Given The Centre "one" should show the following under "Male":
> > +      | Contractual Capacity   | 1000 |
>
> it was mostly a discussion point, perhaps in the creation we should check
> the state?
> which would avoid a load of repetition in our tests
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UKHomeOffice/removals_e2etests/pull/6/files/8d568934e372574dcdb57ebb6c0def7d6fba6f57#r62003591>
>